src/prefabs/creature.js

```diff
@@ -58,9 +59,10 @@ class Creature extends Phaser.Sprite {
 
     update() {
         this.hpText.text = 'HP: ' + this.hp_curr + '/' + this.hp_max;
+        this.game.physics.arcade.collide(this, this.game.world.getByName('Platforms'));
         this.game.physics.arcade.collide(this, this.game.world.getByName('Players'));
         this.game.physics.arcade.collide(this, this.game.world.getByName('Enemies'));
-        this.game.physics.arcade.collide(this, this.game.world.getByName('Platforms'));
+        if(this.busy) this.stop();
```

Comment:
Although it's not only in this file, it may need small refactoring, so it should be made in separate task